lib/routes/admin/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+
+module.exports = function(mediator) {
```

Comment:
Can  you name the function `getAdminRouter` or something to avoid anonymous functions.